app/services/ipam.rb

```diff
@@ -0,0 +1,16 @@
+module IPAM
+  MODES = {:dhcp => N_('DHCP'), :db => N_('Internal DB'), :eui64 => N_('EUI-64'), :none => N_('None')}
```

Comment:
Since now this supports eui-64, can you add a short description to this list? https://github.com/theforeman/foreman/blob/develop/app/views/subnets/_fields.html.erb#L13 

It currently shows up as an option available, but there's no help for it